digital_image_processing/morphological_operations/dilation_operation.py

```diff
@@ -61,14 +63,12 @@ def dilation(image: np.array, kernel: np.array) -> np.array:
     return output
 
 
-# kernel to be applied
-structuring_element = np.array([[0, 1, 0], [1, 1, 1], [0, 1, 0]])
-
-
 if __name__ == "__main__":
     # read original image
-    image = np.array(Image.open(r"..\image_data\lena.jpg"))
-    output = dilation(gray2binary(rgb2gray(image)), structuring_element)
+    lena = np.array(Image.open(Path("../image_data/lena.jpg")))

```

Comment:
This will work even if this Python file is not in the current working directory.
```suggestion
    lena_path = Path(__file__).resolve().parent / "image_data" / "lena.jpg"
    lena = np.array(Image.open(lena_path))
```